### PR TITLE
RPI Lab2

### PR DESCRIPTION
**Pull request by Dmitrytxt**

### CODE REVIEW

File: Lab2.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE HTML>
```
Comment (Dmitrytxt) [inline Lab2.html:1]: White space between pictures (mobile)

File: Lab2.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+<head>
+  <title>Lab2</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="external.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="for-large-screens">
```
Comment (Dmitrytxt) [inline Lab2.html:14]: nav

File: Lab2.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            <ul>
+                <li>
+                    <span>PSD WEBSITES</span>
```
Comment (Dmitrytxt) [inline Lab2.html]: Better to use a

File: Lab2.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            </ul>
+        </div>
+        <div class="for-small-screens">
+            <p id="small">Navigation</p>
+            <i class="material-icons" id="menu">dehaze</i>
+        </div>
+    </header>
+
+    <div class="banner">
+        <p id="caption">Awesome Free PSD </br> templates & Resources</p>
```
Comment (Dmitrytxt) [inline Lab2.html]: h1 instead of p

**Comment by aliaksandrkrasnou** [comment]
Approved, nice 👍

**Pull request closed without merging**